scss/forms/_form-select.scss

```diff
@@ -6,7 +6,7 @@
 .form-select {
   display: block;
   width: 100%;
-  height: $form-select-height;
+  min-height: $form-select-height;
```

Comment:
We can't do this, because FF adds 2px to the height in a pseudo element we can not control: https://searchfox.org/mozilla-central/rev/c7e8bc4996f979e5876b33afae3de3b1ab4f3ae1/layout/style/res/forms.css#339